### ISSUE TITLE
What's up?

### ISSUE THREAD

**Issue by khandelwal** [khandelwal/AskShashank#37]
I am deeply missing @ascott1, @iuloshi, and @stephanieosan. Post your selfies here now. 

Other readers should also participate. Just know that I miss you less. Wait no, that's rude. I miss you too. I maybe just miss Adam more.

**Comment by theresaanna** [comment]
+1000000000000000000000000000000000000000000

**Comment by stephanieosan** [comment]
**I miss you tooooooooooo!** This issue brings me smiles and great happiness.
![selfie-1](http://i.imgur.com/AgMcYJk.gif)

**Comment by theresaanna** [comment]
@stephanieosan Your hair is so loooong now!

**Comment by theresaanna** [comment]
@stephanieosan Also you look like an excited puppy at a window. I mean that as a good thing. Cause @khandelwal will be all "you're saying she looks like a dog?"

**Comment by stephanieosan** [comment]
Yeah! I've been letting it get long in the front, but keeping it short in the back. 
![selfie-4](http://i.imgur.com/nx9qkqk.gif)

I am definitely an excited puppy :smiley:  :heart:

**Comment by theresaanna** [comment]
@stephanieosan I like the angley look. It suits you.

**Comment by iuloshi** [comment]
BIDNESS IN DA FRONT
PARTY IN DA BACK!
IT'S THE SOSAN WAY!! (@stephanieosan) 

can we all go to a conference together soon? preferably in NYC because my conference budget isn't as _robust_ as whence i worked for the govt, which is ironic. 

cc: @contolini @ohsk @mebates @jehlers @clubinski @yuda @theresaanna @ascott1 @khandelwal @errbodyElse

**Comment by contolini** [comment]
I refuse to stay in a hotel with @iuloshi and @theresaanna ever since the "incident".

**Comment by theresaanna** [comment]
@contolini I'm sorry to have broken your heart. I bet that drunken business man was pretty hot.

**Comment by theresaanna** [comment]
@iuloshi From what I can tell, for many of us, at our current employer, conferences are a hard sell. :(

**Comment by ohsk** [comment]
Conference at @contolini's house!

**Comment by contolini** [comment]
@theresaanna To this day I leave the light on for him every night...

**Comment by theresaanna** [comment]
@ohsk He certainly has the space! +100

**Comment by contolini** [comment]
![tumblr_mm6tey6gsn1s7pu0so1_500](https://cloud.githubusercontent.com/assets/1060248/5640999/0eca43ec-95e4-11e4-803e-aacb48c4d6a6.gif)

**Comment by theresaanna** [comment]
Or we can do it up @contolini style and procure a bus and travel the country while conferencing.

**Comment by contolini** [comment]
![spongey](https://cloud.githubusercontent.com/assets/1060248/5641032/cb0cc49e-95e4-11e4-9184-18e7ba4d6833.gif)

**Comment by ascott1** [comment]
HI @khandelwal!

![selfie-3](http://i.imgur.com/SUnvs0r.gif)

**Comment by khandelwal** [comment]
@contolini, @iuloshi, @ohsk Where are your selfies?

**Comment by stephanieosan** [comment]
While there is no excuse for @contolini, perhaps @iuloshi and @ohsk don't have the plugin.

Also, where's YOUR selfie, @khandelwal?

@iuloshi Give us some NYC conf recommendations!

**Comment by jpyuda** [comment]
So when @stephanieosan described her haircut my first reaction was "it's like a reverse mullet!"

But then @iuloshi  beat me to posting that.

But then I was typing that and autocorrect changed "beat" to "berate".

So now I think @iuloshi has to berate me for something.

That is all.

**Comment by jpyuda** [comment]
(Also note: I am not "yuda" on github.)

**Comment by khandelwal** [comment]
BERATE HIM.

**Comment by contolini** [comment]
This is getting weird.

<p>&nbsp;</p>

<p>...</p>

<p>&nbsp;</p>

I like it.

**Comment by theresaanna** [comment]
+10000 to that

**Comment by stephanieosan** [comment]
It's like a lady-mullet. 

You guyyyyys, this thread feels so right!

**Comment by mebates** [comment]
What's up = IMMERSION. New peeps are here on MONDAY! AHHH.

![selfie-1](http://i.imgur.com/2YSw3m8.gif)

**Comment by iuloshi** [comment]
BAHAHAHA SORRY @yuda! WE'RE NORMAL, I SWEAR!

@jpyuda I BETTER SEE YOU AT PRO/DESIGN AT THE END OF THE MONTH!!

@khandelwal How's this live feed of me working right meow?
![cfpb](https://cloud.githubusercontent.com/assets/1676307/5656108/b5787b66-96a5-11e4-8060-202dae172a30.gif)
